delivery_auto_refresh/models/sale_order.py

```diff
@@ -145,9 +211,25 @@ class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
     def _get_protected_fields(self):
-        """Avoid locked orders validation error when voiding the
-        delivery line"""
+        # Avoid locked orders validation error when voiding the delivery line
```

Comment:
Why changing legit docstring to comments? The docstring is specifying what the method override is doing, so please don't change it.

Reply:
Wow, that changes my way of thinking in all of this, but it's also a pity as I feel that a comment doesn't cover the feature I try to put: to explain why the whole method is being overridden. Do you know any other way to put this?